civis/io/_tables.py

```diff
@@ -1033,6 +1033,9 @@ def civis_file_to_table(file_id, database, table, client=None,
     if client is None:
         client = APIClient()
 
+    if type(file_id) == str:
+        raise TypeError('Invalid type for file_id: str')
```

Comment:
Could the error message also mention what the data type should be (i.e., either an int or an iterable of int), in addition to saying it cannot be a string?

Also, would you be able to add this `TypeError` to the "Raises" section in the docstring of this function?

Reply:
Will do, I didn't mention the data type allowed b/c I figured if the allowed types were ever changed it would need to be changed in the error message too.